### PR TITLE
Masquerade Clothing Expansion

### PR DESCRIPTION
**Pull request by Croissant-Man**
[SPRITING][CLOTHING]
## About the PR 
This PR adds suit and gender neutral outfit counterparts to the masquerade themed dresses, as well as a cargo crate to get them from. note that these outfits aren't going to be added to the clothing booth until the ongoing rework for it is done
image
(masks come separately) 

## Why's this needed? 
Gentlemen and gentlethems should be able to be fashionably mysterious too

## Changelog 
```changelog
(u)Croissant_Man1
(*)Added Masquerade suits and outfits to the game
(+)Added a cargo crate with new and old masquerade gear to cargo so you can access the new clothes/prepare a masquerade ball
(+)Changed the old "Masquerade Crate" to be named "Mask Crate" since that and the masquerade gear are very different
```

### CODE REVIEW

**Comment by wolfolotl** [comment]
love the concept, but the worn sprites are pretty busy. the gender neutral variants are especially hard to parse, visually speaking - the original dresses are also a bit busy for my liking, but the more constructed nature of these garments makes the read even less clear

**Comment by Croissant-Man** [comment]
Ah, generally speaking the feedback on these I got on the discord server were very positive but I see what you’re saying.
the neutral clothes are intentionally made to closely resemble the dresses, and are meant to be like a one-piece outfit, the peacock outfit especially reminding me of something like an ice skating outfit I saw once before.

**Comment by Croissant-Man** [comment]
Woah look I made this outfit more like a solid object and less like a mess of petals
image

**Comment by Croissant-Man** [comment]
most up to date sprite on that thingy.
image

**Comment by Croissant-Man** [comment]
most up to date sprite of the one I've fixed.
image

**Comment by Croissant-Man** [comment]
PR updated with the new sprites, and a human model to show everything off properly.

**Review by WALPVRGIS — CHANGES_REQUESTED** [review]
I can see why you've chosen to use that dithered texture as it's on the dresses these are based off of, but it looks quite chaotic and hard to read on these outfits. The dithering on the dresses represents a ruffled texture (I assume, I didn't sprite the originals), and I don't think it translates effectively into these outfits.

Imo I think you could achieve making options that match in palette and are much more readable without the texture! I'm also confused at what's happening with the peacock suit, I can't parse what the various parts are. I think we should avoid outfits that aren't easy to parse, as we then run into the issue of players looking too indistinct.

**Comment by Croissant-Man** [comment]
So, if i'm getting your meaning right its mostly the peacock outfits in general that need changing, right?

**Comment by Croissant-Man** [comment]
putting this project down for now. i'll probably pick it back up if i come up with interesting designs to replace the sprites that needed to be replaced.

**Comment by Croissant-Man** [comment]
re-opening so as to remind myself that i want to complete this PR at some point.
